File: Data Structures & Algorithms/duplicate-integer/submission-14.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        boolean value = false;
        Set set = new Set ();
        for int : i in nums {
            if set.contains(i) {
                return true;
            }
            set.add(i);
        }
        return false;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-14.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        myMap = {}
        
        for i,n in enumerate(nums) :
            complement = target - n
            if complement in myMap :
                return [myMap[complement],i]
            return myMap[n]
